### PR TITLE
RMC-335 Telephone Capture SPG

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
New feature to request telephone capture for SPG unit and establishment level cases.

# What has changed
* Add scenarios for SPG telephone capture requests

# How to test?
Build and run with the linked branch of case API

# Links
https://trello.com/c/CzO4dcfw/501-telephone-capture-spg-unit-and-estab-level-3
https://github.com/ONSdigital/census-rm-case-api/pull/48

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[Telephone Capture: SPG - unit and estab level (3)](https://trello.com/c/CzO4dcfw/501-telephone-capture-spg-unit-and-estab-level-3)

**Comment by AdamHawtin** [comment]
[Telephone Capture: SPG - unit and estab level (3)](https://trello.com/c/CzO4dcfw/501-telephone-capture-spg-unit-and-estab-level-3)

File: acceptance_tests/features/telephone_capture.feature [hunk 1]
```diff
@@ -5,18 +5,22 @@ Feature: Telephone capture
...
     Examples:
-      | sample file                  | case type | country code | questionnaire type |
-      | sample_1_english_unit.csv    | HH        | E            | 01                 |
-      | sample_1_welsh_unit.csv      | HH        | W            | 02                 |
-      | sample_1_ni_unit.csv         | HH        | N            | 04                 |
-      | sample_1_english_CE_unit.csv | CE        | E            | 21                 |
-      | sample_1_welsh_CE_unit.csv   | CE        | W            | 22                 |
-      | sample_1_ni_CE_unit.csv      | CE        | N            | 24                 |
+      | sample file                    | address level | case type | country code | questionnaire type |
+      | sample_1_english_unit.csv      | U             | HH        | E            | 01                 |
```
Comment (liamtoozer) [inline acceptance_tests/features/telephone_capture.feature]: Might be out of scope for this ticket but is it worth adding in `HH` to the file names now that we're distinguishing things like SPC and CEs now?

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

File: acceptance_tests/features/telephone_capture.feature (hunk 1 above)
Comment (AdamHawtin) [reply acceptance_tests/features/telephone_capture.feature]: Yeah, worth renaming them now.

**Review by gavinleeedwards — APPROVED** [review]

**Pull request merged**